.gitignore

```diff
@@ -103,6 +103,9 @@ ENV/
 .mypy_cache/
 
 .vscode/
-test_db.sqlite
+*.sqlite
+
+# pipenv files
+Pipfile*
```

Comment:
I think you could just leave these for someone that uses the `pipenv` workflow.

Reply:
Done. I have removed them from gitignore.